testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -59,6 +61,13 @@ public void launch(ILaunchConfiguration configuration, String mode, ILaunch laun
           new String[] { install.getId() }), null,
           IJavaLaunchConfigurationConstants.ERR_VM_RUNNER_DOES_NOT_EXIST);
     }
+    AbstractVMInstall vmi = (AbstractVMInstall) install;
+    String vmver = vmi.getJavaVersion();
+    if (vmver.compareTo("1.7") < 0) { //$NON-NLS-1$

```

Comment:
aha, right, but i really don't know about the future version.
i think we can just leave it and incorporate with the future version in future